client/my-sites/themes/theme-showcase.jsx

```diff
@@ -348,7 +348,7 @@ class ThemeShowcase extends React.Component {
 									<hr />
 								) : (
 									<Button onClick={ this.toggleShowcase } data-e2e-value="open-themes-button">
-										{ translate( 'Show All Themes' ) }
+										{ translate( 'Show all themes' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.